tools/scripts/ansible/setup-playbook.yml

```diff
@@ -1,4 +1,8 @@
 - import_playbook: server-playbook.yml
+- import_playbook: mysql-playbook.yml
+- import_playbook: nginx-playbook.yml
+- import_playbook: tomcat-playbook.yml
+- import_playbook: cldr-apps-playbook.yml
 - import_playbook: packages-playbook.yml
 - import_playbook: certbot-playbook.yml
 - import_playbook: hostname-playbook.yml
```

Comment:
@btangmu it's the order of execution.  You have 'server playbook` before `nginx playbook`, so it tries to update `/var/www/` before nginx is actually installed.

So this is a valid error. putting `nginx-playbook.yml` first seemed  to help, but please consider the order

Reply:
Thanks! I'll work on it